atac.wdl

```diff
@@ -924,35 +1067,51 @@ task trim_adapter {
 	}
 }
 
-task bowtie2 {
-	File idx_tar 		# reference bowtie2 index tar
+task align {
+	String aligner
+	String mito_chr_name
+	File chrsz			# 2-col chromosome sizes file
+	File? custom_align_py
+	File idx_tar		# reference index tar
 	File? fastq_R1 		# [read_end_id]
 	File? fastq_R2
 	Boolean paired_end
 	Int multimapping
-	String bowtie2_param_se
-	String bowtie2_param_pe
+
 	Int cpu
 	Int mem_mb
 	Int time_hr
 	String disks
 
 	command {
-		python $(which encode_bowtie2.py) \
-			${idx_tar} \
-			${fastq_R1} ${fastq_R2} \
-			${if paired_end then "--paired-end" else ""} \
-			${"--multimapping " + multimapping} \
-			--bowtie2-param-se ' ${bowtie2_param_se}' \
-			--bowtie2-param-pe ' ${bowtie2_param_pe}' \
+		if [ "${aligner}" == "bowtie2" ]; then
+			python3 $(which encode_task_bowtie2.py) \
+				${idx_tar} \
+				${fastq_R1} ${fastq_R2} \
+				${if paired_end then "--paired-end" else ""} \
+				${"--multimapping " + multimapping} \
+				${"--nth " + cpu}
+		else
+			python ${custom_align_py} \
```

Comment:
Does this mean the custom aligner script runs in a different environment than bowtie?

Reply:
thank, changed it to python3